src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.Simple.cs

```diff
@@ -473,12 +484,14 @@ public void OperationsOnUnconnectedClient()
         {
             using (NamedPipePair pair = CreateNamedPipePair())
             {
+                pair.Connect();
```

Comment:
This seems like two separate but very similar scenarios:

1. Make a connected pair, dispose the client, then try to perform operations on it.
2. Make a clientstream, dispose it, then try to perform operations on it.

I suppose, though, that the behavior is the same in either case and the connected case is more specific so we should be fine to just use that one. For example, pipe.Connect() throws ObjectDisposedException in either case but it would throw InvalidOperationException if it wasnt disposed but was connected, so having the connected case shows that ObjectDisposedException is going to be thrown before InvalidOperationException.